src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,25 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  justify-content: center;
+  gap: 46px 48px;
+  padding: 50px 40px;
+}
+
+@media (min-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .catalog {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .catalog {
+    grid-template-columns: repeat(4, 200px);
```

Comment:
For a screen of 1024px width, you may want to allow for more flexibility in the number of columns. Using 'auto-fill' or 'auto-fit' with a minmax function can help create a more responsive design.